annotation/annotation/gradle.properties

```diff
@@ -14,5 +14,5 @@
 # limitations under the License.
 #
 
-artifactRedirecting.publication.targetNames=jvm,macosX64,macosArm64,uikitX64,uikitArm64,uikitSimArm64,linuxX64
+artifactRedirecting.publication.targetNames=jvm,macosX64,macosArm64,uikitX64,uikitArm64,uikitSimArm64,watchosArm64,linuxX64,linuxArm64,mingwX64,js
```

Comment:
should we add wasmJs too? 

js variant is available only with 1.9.1  - https://maven.google.com/web/index.html#androidx.annotation:annotation-js

Did you want to update the version here: https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/gradle.properties#L117 ?

Reply:
> wasmJs was added earlier.

Missed that. We should.

> Did you want to update the version here:

Yes! I updated it in `integration` branch but a script ignored it. I'll copy this change manually

Reply:
Done